build-image/README.md

```diff
@@ -0,0 +1,44 @@
+# Verifiable builds using Docker
+
+Docker image based on our base CI image `<base-ci-linux:latest>`.
+
+Used for reproducible builds in `cargo contract build --verifiable`
+
+**Rust versions:**
+
+Currently, the 1.69 toolchain is temporarily required to build ink! contracts because of https://github.com/paritytech/cargo-contract/issues/1139
+
+**Rust tools & toolchains:**
+
+We use stable releases from crates.io
+
+- `cargo-contract`
+- `wasm32-unknown-unknown`: The toolchain to compile Rust codebases for Wasm.
+
+[Click here](https://hub.docker.com/repository/docker/paritytech/contracts-verifiable) for the registry.
+
+## Usage
+
+The default entry point is `ENTRYPOINT [ "cargo", "contract", "build", "--release" ]`
+and work directory is set to `/contract`. You need to mount the folder with the contract to the container.
+
+```bash
+docker run -d \
+    --name ink-container \
+    --mount type=bind,source="$(pwd)",target="/contract" \
+    paritytech/contracts-verified
+```
+
+If you have multi-contract project:
+```
+my-app/
+├─ ink-project-a/
+│  ├─ Cargo.toml
+│  ├─ lib.rs
+├─ ink-project-b/
+│  ├─ Cargo.toml
+│  ├─ lib.rs
+├─ rust-toolchain
+```
+Make sure you mount `my-app` directory and then call
```

Comment:
```suggestion
Make sure to run the tool in `my-app` directory
```
I don't think the readers of this guide need to know about _mounting_. After all, for the final consumers of `cargo-contract` we don't want them to need to know about docker inside.

Reply:
My belief is if people want to run the container manually then they know what they are doing, and, in that case, we should be precise in the terminology, because there are multiple ways to _run the tool in directory_

Reply:
But the command you give below is for `cargo contract` and not docker. Which means these people don't even know they're running container manually.